docs/import/create_score_asset.js

```diff
@@ -22,16 +22,20 @@ function countDamageAndBuildings(feature, damage, buildings) {
   const geoId = feature.get(geoidTag);
   const totalBuildings = ee.Algorithms.If(
       buildings.contains(geoId), buildings.get(geoId), ee.Number(0));
+  const rawIncomeValue = ee.String(feature.get(incomeTag));
+  const income = ee.Algorithms.If(
+      ee.Algorithms.IsEqual(rawIncomeValue, ee.String('250,000+')),
+      ee.Number(250000),
+      ee.Algorithms.If(
+          ee.Algorithms.IsEqual(rawIncomeValue, ee.String('-')), null,
```

Comment:
how do we handle null values when rendering a layer? Is that a follow-up? Also, how do we handle it in pop-up display?

Reply:
Before it was just rendering as white since null < all numbers but now we're setting it to be transparent if the val is null. Probably a check to have to prevent referencing null in the discrete case as well (since the color map is value -> color).